src/frontends/pytorch/src/frontend.cpp

```diff
@@ -182,6 +183,8 @@ void FrontEnd::normalize(const std::shared_ptr<ov::Model>& model) const {
     manager.register_pass<ov::frontend::pytorch::pass::AtenCatToConcat>();
     manager.register_pass<ov::frontend::pytorch::pass::AppendListUnpackReplacer>();
     manager.register_pass<ov::frontend::pytorch::pass::AtenStackListConstructReplacer>();
+    manager.register_pass<ov::pass::MarkDequantizationSubgraph>(
```

Comment:
Why do we need this transformation here 2nd time? We already have it on line 177

Reply:
We have produced new sub-graph with constant->convert in `AtenStackListConstructReplacer` that should be preserved.

Reply:
But it needs to be protected only against `ConstantFolding` and next one will be called in moc transformations which also have `MarkDequantizationSubgraph` transformation before `ConstantFolding`, so it should be safe to remove it from here.